app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -64,21 +69,38 @@ class SettingsRobot {
 
     fun verifyRateOnGooglePlay() = assertRateOnGooglePlay()
     fun verifyAboutFirefoxPreview() = assertAboutFirefoxPreview()
+    fun verifyGooglePlayRedirect() = assertGooglePlayRedirect()
 
     class Transition {
 
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
 
         fun goBack(interact: HomeScreenRobot.() -> Unit): HomeScreenRobot.Transition {
-            mDevice.waitForIdle()
-            goBackButton().perform(ViewActions.click())
+            goBackButton().click()
 
             HomeScreenRobot().interact()
             return HomeScreenRobot.Transition()
         }
 
-        fun openSearchSubMenu(interact: SettingsSubMenuSearchRobot.() -> Unit): SettingsSubMenuSearchRobot.Transition {
-            mDevice.waitForIdle()
+        fun clickOnHelpButton(interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
+            assertHelp().click()
+
+            BrowserRobot().interact()
+            return BrowserRobot.Transition()
+        }
```

Comment:
Why is the Help button interaction in this robot? Shouldn't it be part of the three dot menu robots?

Reply:
It definitely should be. I think there was a Help option at the time when this was written but has now been removed from the settings screen.